members/index.html

```diff
@@ -0,0 +1,174 @@
+---
+layout: default
+title: Members
+description: WICS Members
+---
+
+<link rel="stylesheet" type="text/css" href="members.css" />
+
+
+<div class="container">
+    <div class="title">
+        <b>WICS Members</b>
+    </div>
+</div>
+
+
+{% include_relative modal.html %}
+<div class="container">
+        {% assign img_dir = site.baseurl | append: '/assets/img/resources-imgs' %}
+
+        <div class="execs">
```

Comment:
Let's change these to an `id` rather than a `class` . I believe the `class` attribute is used more often for styling purposes 😸

Reply:
good catch